README.md

```diff
@@ -5,6 +5,64 @@
 [![Code Coverage](https://codecov.io/gh/AlgebraicJulia/AlgebraicOptimization.jl/branch/main/graph/badge.svg)](https://codecov.io/gh/AlgebraicJulia/AlgebraicOptimizatione.jl)
 [![CI/CD](https://github.com/AlgebraicJulia/AlgebraicOptimization.jl/actions/workflows/julia_ci.yml/badge.svg)](https://github.com/AlgebraicJulia/AlgebraicOptimization.jl/actions/workflows/julia_ci.yml)
 
-Building and solving optimization problems compositionally.
+This package is designed for building large optimization problems out of simpler subproblems and automatically compiling them to a distributed solver.
+
+## Basic Usage
+
+The most simple use of this package is making and solving a composite optimization problem. For example, say we have three subproblems whose objectives are given by some random quadratic cost functions:
+```julia
+using AlgebraicOptimization
+using Catlab
+
+# Problem parameters.
+P = rand(-1:0.01:1,5,5)
+P = P'*P
+Q = rand(-1:0.01:1,3,3)
+Q = Q'*Q
+R = rand(-1:0.01:1,4,4)
+R = R'*R
+
+a = rand(-1:0.01:1,5)
+b = rand(-1:0.01:1,3)
+c = rand(-1:0.01:1,4)
```

Comment:
using random functions in docs is rough, because they will be different in every build. Better to use rand to get values and then paste them in. Also you will want to truncate to 1-4 sigfigs